vmhost/contexts/storage_test.go

```diff
@@ -462,25 +558,17 @@ func TestStorageContext_GetStorageFromAddress(t *testing.T) {
 			return nil, nil
 		},
 		GetStorageDataCalled: func(accountsAddress []byte, index []byte) ([]byte, uint32, error) {
-			return internalData, 0, nil
+			return internalData, 0, getStorageErr
 		},
 	}
+}
 
-	storageCtx, _ := NewStorageContext(host, bcHook, reservedTestPrefix)
-	storageCtx.SetAddress(scAddress)
-
-	key := []byte("key")
-	data, _, err := storageCtx.GetStorageFromAddress(scAddress, key)
-	require.Nil(t, err)
-	require.Equal(t, data, internalData)
-
-	data, _, err = storageCtx.GetStorageFromAddress(readable, key)
-	require.Nil(t, err)
-	require.Equal(t, data, internalData)
+func TestStorageContext_LoadGasStoreGasPerKey(t *testing.T) {
+	// TODO
```

Comment:
when ?

Reply:
I need to sync with @BeniaminDrasovean, but it's likely going to be my next PR.